### PR TITLE
Encrypter les donnees

### PR DESCRIPTION
**Pull request by almet**
Supersedes #61. This version has the script directly in the blogpost and is rebased on master.

### CODE REVIEW

**Comment by michielbdejong** [comment]
Great, thanks a lot!! I removed the references to github, and switched back to latest Kinto.js master (this code does not work with 1.0-rc3). Do we maybe want to wait with publishing this blogpost until 1.0-rc4 is out?

**Comment by almet** [comment]
I think we're good to let it as-is.

**Comment by Natim** [comment]
Do you want to keep the sorry for writting in English? I don't think it is necessary.

File: content/2015.09.kinto-encryption-example.rst [hunk 1]
```diff
@@ -0,0 +1,228 @@
...
+        encode,
+        decode
+      };
+    };
+
+We create two Kinto instances, so that we can test syncing from one instance
+to the other, via the public Kinto demo instance on AWS. Creating multiple Kinto
+instances within the same origin is now possible with the recently added
+``dbPrefix``
+option:
```
Comment (Natim) [inline content/2015.09.kinto-encryption-example.rst]: no need for a new line here.

**Comment by Natim** [comment]
Published http://www.servicedenuages.fr/en/kinto-encryption-example thanks a lot.

**Comment by michielbdejong** [comment]
\o/ Thanks for all your help!

**Pull request merged**